### ISSUE TITLE
Not able to move to first option of the suggested option list  from the  last option on pressing up arrow key in Picker

### ISSUE THREAD

**Issue by HarkeshS** [OfficeDev/office-ui-fabric-react#11696]
Not able to move to first option of the suggested option list  from the  last option on pressing up arrow key in Picker

### Environment Information

OS: Win 10
Browser:  Chrome
### Describe the issue:

Not able to move to first option of the suggested option list  from the  last option on pressing up arrow key in Picker

### Please provide a reproduction of the issue in a codepen:
https://developer.microsoft.com/en-us/fabric#/controls/web/pickers
Code pen--https://codepen.io/anubhuti14/pen/povKmzW?editable=true

#### Actual behavior:

Not able to move to first option of the suggested option list  from the  last option on pressing up arrow key in Picker

#### Expected behavior:

should be able to move to first option of the suggested option list  from the  last option on pressing up arrow key in Picker

**Comment by xugao** [comment]
@HarkeshS - I checked and it's unclear what the bug/issue is here. could you please provide more detailed step on how to repro this issue?

**Comment by HarkeshS** [comment]
Hi @xugao,

Repro steps
1. First search any suggestion like pressing b char in textbox
2. Now move at end of element in list by pressing down key
3. When at end of element, press down key once
4 Now press UP key and see the issue, You will not able to cursor in list when you press UP key.
![ReproImage](https://user-images.githubusercontent.com/53938674/72405234-28509c80-377e-11ea-9d3e-6b0fbe37e37d.png)
Regards,
Harkesh

**Comment by xugao** [comment]
I wonder why the focus goes to input at

**Comment by xugao** [comment]
@HarkeshS - agree this should not be intended. just to confirm, you expect pressing UP key from input text goes back to  *last*  option in the dropdown list, right?

**Comment by xugao** [comment]
@HarkeshS  - I looked into this a bit more. Since the focus is on the input after pressing DOWN from last option, when you the pressing UP with focus on the input, the **expected** behavior is to move focus to the beginning of the text. And in that case, you will need to press DOWN to cycle through the options.

**Comment by HarkeshS** [comment]
Hi @xugao,
When you pressing DOWN from last option it should not go to INPUT as you mentioned. So it should be issue.

**Comment by xugao** [comment]
@HarkeshS - I am pretty sure this behavior is by design. i will confirm with my team and get back to you

**Comment by xugao** [comment]
pressing DOWN from last option it should not go to INPUT. will look into fixing
